src/lib/registration/ui/migration_repos_workflow.rb

```diff
@@ -563,13 +563,24 @@ def update_releasever
       end
 
       # check the system status at upgrade and return the symbol for the next step
-      # @return [Symabol] workflow symbol, :skip => do not use the SCC/SMT/RMT upgrade
+      # @return [Symbol] workflow symbol, :skip => do not use the SCC/SMT/RMT upgrade
       #   (unregistered system or explicitly requested by user), :next =>
       #   continue with the SCC/SMT/RMT based upgrade
       def system_upgrade_check
         log.info "System upgrade mode detected"
         # media based upgrade requested by user
         if Yast::Linuxrc.InstallInf("MediaUpgrade") == "1"
+          # we cannot do medium based upgrade using the Online medium,
+          # it does not contain any packages
+          if Y2Packager::MediumType.online?
+            log.warn "The Online medium cannot be used with the media_upgrade=1 boot option"
```

Comment:
warn or error?

Reply:
We should use warnings for not critical issues or user errors. There is an openQA test which scans the y2log for error after the installation.